sdk/extensions/wcf/Microsoft.CoreWCF.Azure.StorageQueues/README.md

```diff
@@ -0,0 +1,79 @@
+# CoreWCF Azure Queue Storage library
+
+CoreWCF Azure Queue Storage is the service side library that will help existing WCF services to be able to use Azure Queue Storage to communicate with clients as a modern replacement to using MSMQ.
+
+## Getting started
+
+### Install the package
+
+Install the Microsoft.CoreWCF.Azure.StorageQueues library for .NET with [NuGet][nuget]:
+
+```dotnetcli
+dotnet add package Microsoft.CoreWCF.Azure.StorageQueues
+```
+
+### Prerequisites
+
+You need an [Azure subscription][azure_sub] and a
+[Storage Account][storage_account_docs] to use this package.
+
+To create a new Storage Account, you can use the [Azure Portal][storage_account_create_portal],
+[Azure PowerShell][storage_account_create_ps], or the [Azure CLI][storage_account_create_cli].
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+```
+
+### Authenticate the service to Azure
+
+In order to receive requests from the Azure Queue Storage service, you'll need to configure CoreWCF with the appropriate endpoint and credentials.  The [Azure Identity library][identity] makes it easy to add Azure Active Directory support for authenticating with Azure services.
+
+```C# Snippet:CoreWCF_Azure_Storage_Queues_Sample_DefaultAzureCredential
+app.UseServiceModel(services =>
```

Comment:
It would be helpful here to show the method wrapping this - particularly where `app` and `services` come from.

Reply:
How that code looks varies depending on if you are using the older asp.net core 2.1 style (basically pre 6.0) of creating a service, or if you are using the newer minimal API style of service creation. The snippet here is agnostic to whichever style you are using. Picking one over the other could cause confusion for whoever is using the style we don't provide. CoreWCF is middleware to be used with asp.net core and we're trying to avoid being a tutorial on how to use asp.net core.

Reply:
Fair enough.